### PR TITLE
feat(linkedin): init

### PR DESCRIPTION
**Pull request by isabelroses**
## 🎉 Theme for [LinkedIn](linkedin.com/) 🎉

Social networking site

## 🗒 Checklist 🗒

- [x] I have read and followed Catppuccin's [submission guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/userstyle-creation.md).
- [x] I have made a new directory underneath `/styles/<name-of-website>` containing the contents of the [`/template`](https://github.com/catppuccin/userstyles/blob/main/template/) directory.
  - [x] I have ensured that the new directory is in **lower-kebab-case**.
  - [x] I have followed the template and kept the preprocessor as [LESS](https://lesscss.org/#overview).
- [x] I have made sure to update the
      [`userstyles.yml`](https://github.com/catppuccin/userstyles/blob/main/src/userstyles.yml)
      file with information about the new userstyle.
- [x] I have included the following files:
  - `catppuccin.user.css` - all the CSS for the userstyle, based on the
    template.
  - `assets/mocha.webp`, `assets/macchiato.webp`, `assets/frappe.webp` and
    `assets/latte.webp` - individual screenshots of the themed website, in all 4
    Catppuccin flavors.
  - `catwalk.webp` - image of all individual screenshots stitched together,
    generated via [catwalk](https://github.com/catppuccin/toolbox#-catwalk).

### CODE REVIEW

**Comment by sgoudham** [comment]
Thanks for working on this @isabelroses :raised_hands: 
Just commenting what I mentioned on discord, the contrast for notifications is very harsh on latte.

E.g.
linkedin

For reference, here is what the default light mode looks like
linkedin

**Comment by isabelroses** [comment]
> The contrast for notifications is very harsh on latte.

As of https://github.com/catppuccin/userstyles/pull/277/commits/fc89fe985626950eea9f100f81a06a88cc86007f, I have fixed some of the natively faded out, or similar.

File: src/userstyles.yml [hunk 1]
```diff
@@ -269,6 +269,13 @@ userstyles:
       app-link: "https://github.com/libreddit/libreddit"
       current-maintainers: [*isabelroses]
       past-maintainers: [*andreasgrafen]
+  linkedin:
+    name: LinkedIn
+    category: social
+    icon: LinkedIn
+    readme:
+      app-link: "https://www.linkedin.com"
+      current-maintainers: [*isabelroses]
```
Comment (nekowinston) [inline src/userstyles.yml:278]: Please add @sgoudham to the current maintainers

**Review by nekowinston — CHANGES_REQUESTED** [review]

File: src/userstyles.yml (hunk 1 above)
Comment (sgoudham) [reply src/userstyles.yml:278]: Please **do not**

**Review by sgoudham — APPROVED** [review]
Thanks for doing this :D

**Pull request merged**